### ISSUE TITLE
Error when running Create Table Script

### ISSUE THREAD

**Issue by dschinkel** [oguimbal/pg-mem#6]
In **pgAdmin** I generate a Create Table Script off my table then try to use it with your lib:

```
const pg = newDb().adapters.createPgPromise();
const client = await pg.connect();
const crafterTable = await client.none(`CREATE TABLE public.crafter
(
    crafter_name_first character varying COLLATE pg_catalog."default" NOT NULL,
    crafter_name_last character varying COLLATE pg_catalog."default" NOT NULL,
)

TABLESPACE pg_default;

ALTER TABLE public.crafter
    OWNER to postgres;

GRANT ALL ON TABLE public.crafter TO postgres;

GRANT SELECT ON TABLE public.crafter TO wedotdd_readonly_crafter_id_fistname_lastname;

GRANT SELECT(crafter_name_first) ON public.crafter TO wedotdd_readonly_crafter_id_fistname_lastname;

GRANT SELECT(crafter_name_last) ON public.crafter TO wedotdd_readonly_crafter_id_fistname_lastname;

GRANT SELECT(crafter_id) ON public.crafter TO wedotdd_readonly_crafter_id_fistname_lastname;`);
```

**but, I end up with errors like this, not sure how to resolve this, or if your lib can handle this at all?:**
```
Error: Syntax error at line 4 col 41:

      crafter_name_last character varying COLLATE
                                          ^
Unexpected kw_collate token: "COLLATE". Instead, I was expecting to see one of the following:


A lparen token based on:
    lparen →  ● %lparen
    data_type$ebnf$1$subexpression$1 →  ● lparen int rparen
    data_type$ebnf$1 →  ● data_type$ebnf$1$subexpression$1
    data_type → data_type_simple ● data_type$ebnf$1 data_type$ebnf$2
    createtable_column → word ● data_type createtable_column$ebnf$1 createtable_column$ebnf$2
    createtable_declaration$subexpression$1 →  ● createtable_column
    createtable_declaration →  ● createtable_declaration$subexpression$1
    createtable_declarationlist$ebnf$1$subexpression$1 → comma ● createtable_declaration
    createtable_declarationlist$ebnf$1 → createtable_declarationlist$ebnf$1 ● createtable_declarationlist$ebnf$1$subexpression$1
    createtable_declarationlist → createtable_declaration ● createtable_declarationlist$ebnf$1
    createtable_statement → %kw_create %kw_table createtable_statement$ebnf$1 word lparen ● createtable_declarationlist rparen
    statement →  ● createtable_statement
    main → main$ebnf$1 ● statement main$ebnf$2 main$ebnf$3

```

**Comment by oguimbal** [comment]
Yup. Collate is not implemented.
I should have some time today, I'll implement it right away (should not take long).

That said, I'm only planning to implement parsing of "collate" statements. Not the actual collation stuff (which means that it's OK as long as you dont use "default" collation, but you **could** see some beahviour changes between pg & pg-mem if you use something else).

**Comment by dschinkel** [comment]
Ok.  Yea so part of this goes back to me being able to auto generate scripts of my schema and having your in-memory DB run these scrips so that I always have an up-to-date version of my prod database.  But when you create scripts from pgAdmin of course it includes all this as part of the script so...not sure how I can strip any of that out automatically, so I don't see how I can automate this if yours won't run the scripts gened by postgres, you know what I mean?

**Comment by oguimbal** [comment]
Closed as kind-of-duplicate of #5 :)

**Comment by dschinkel** [comment]
Thanks very much for [How to import my production schema in pg-mem](https://github.com/oguimbal/pg-mem/wiki/FAQ#-how-to-import-my-production-schema-in-pg-mem-).   It's awesome documentation you added and helps.

The only thing that's missing though which I really need to be added to this in order for me to be able to use your lib is **migration support / API**.  I can't really use this to the fullest if I can't run migrations on .sql files much like you can with [node-sqlite](https://github.com/kriasoft/node-sqlite#migrations)
